Rakefile

```diff
@@ -3,3 +3,23 @@ require "logstash/devutils/rake"
 task :default do
   system('rake -T')
 end
+
+task :gradle => "gradle.properties" do
+  system("./gradlew vendor")
```

Comment:
Can you apply [this](https://github.com/logstash-plugins/logstash-input-beats/commit/b64c5cbcb0c94edc7147e140caccfb603fa7b677) change here too ? (It protects against releasing a bad gem due to a failure here)